app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,49 @@
+footer {
+  display: flex;
+  align-items: center;
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  height: 200px;
+  border-radius: 15px;
+  margin: 0 auto;
+  margin-bottom: 15px;
+  padding: 15px;
```

Comment:
Can you change the background-color to lightgrey and the border-radius to 20px to fit the current design ?

And margin + margin-bottom can be changed to `margin: 0 auto 15px;`

Try adding `justify-content: space-around;`
Then you can remove the margin 0 auto of the icon and the description